pkg/templates/charts/toggle/console/templates/console-plugin.yaml

```
@@ -1,24 +1,24 @@
 # Copyright Contributors to the Open Cluster Management project
-
-apiVersion: console.openshift.io/v1alpha1
+apiVersion: console.openshift.io/v1
 kind: ConsolePlugin
 metadata:
   name: acm
-  annotations:
-    console.openshift.io/use-i18n: "true"
 spec:
   displayName: Red Hat Advanced Cluster Management
+  backend:
+    service:
+      basePath: /plugin/
+      name: console-chart-console-v2
+      namespace: {{ .Values.global.namespace }}
+      port: 3000
```

For the `consolePlugin`, the `spec.backend.type` value is required. If not added the YAML will be considered invalid.
```suggestion
  backend:
    service:
      basePath: /plugin/
      name: console-chart-console-v2
      namespace: {{ .Values.global.namespace }}
      port: 3000
    type: Service
```